### PR TITLE
[openweathermap] Internationalization of discovery results

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

File: bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_fr.properties [hunk 1]
```diff
@@ -257,5 +257,5 @@ offline.conf-error-not-supported-onecall-number-of-hours = Le paramètre 'foreca
 offline.conf-error-not-supported-onecall-number-of-days = Le paramètre 'forecastDays' doit être compris entre 0 et 7.
 
 # discovery result
-discovery.openweathermap.weather-and-forecast.api.local.label = Météo locale et prévisions
-discovery.openweathermap.uvindex.api.local.label = Indice UV local
+discovery.weather-and-forecast.local.label = Météo locale et prévisions
+discovery.air-pollution.local.label = Indice UV local
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_fr.properties:261]: Can you verify if this translation is correct? If not remove it from the French properties and let it be translated via Crowdin.

**Review by cweitkamp — CHANGES_REQUESTED** [review]
Thanks. Looks much better and will probably work if someone does not choose id `api` for his bridge.

File: bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_fr.properties (hunk 1 above)
Comment (lolodomo) [reply bundles/org.openhab.binding.openweathermap/src/main/resources/OH-INF/i18n/openweathermap_fr.properties:261]: Translation is correct.

**Comment by lolodomo** [comment]
> Looks much better and will probably work if someone does not choose id `api` for his bridge

Yes, that is the purpose and of course I tested it.

**Review by cweitkamp — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**